specs/MultiProfile.md

```diff
@@ -328,6 +362,15 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
+  /// The render process of webviews on this profile will asynchronously exit with the reason `COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`. See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED' for more details.
```

Comment:
If the host app wants to delete the profile before exiting, what happens if the host app calls Profile.Delete and then immediately closes all their CoreWebView2Controller objects without waiting for the browser process to exit? Will that potentially interrupt the profile deletion and the profile doesn't get deleted? If so, we should document what they need to wait for to ensure that profile deletion has completed.

Reply:
Do not need to wait for to ensure that profile deletion has completed. Because The Delete API will send MOJOM message, and CoreWebView2Controller.Close will send MOJOM message too. They are in order, first come first.